website/docs/cdktf/debugging.mdx

```diff
@@ -0,0 +1,59 @@
+---
+page_title: Debugging
+description: >-
+  Learn how to troubleshoot problems in your CDKTF application.
+---
+
+# Debugging
+
+## Increasing the verbosity
+
+You can get more information about what CDKTF is doing and what problems have occured by setting the `CDKTF_LOG_LEVEL` environment variable to `debug`. If you want the logs written to a file you can set `CDKTF_LOG_FILE_DIRECTORY` to a file path, a `cdktf.log` file with all logs will be written. This can be helpful if you need to share them in e.g. a bug report.
+
+## Troubleshooting
+
+To understand where the problem is coming from it's a good first step to see if it's a synth-time issue or an execution-time issue.
+If `cdktf synth` fails you have a synth-time issue, otherwise it's an execution-time issue.
```

Comment:
I actually wonder if we even need the nesting here --> What if we just had synth time and execution time issues as Level 2 headings. 

## Logging

## Synth Issues

## Execution Issues